samples/nrf9160/https_client/README.rst

```diff
@@ -40,6 +40,16 @@ This certificate is provided in the :file:`samples/nrf9160/https_client/cert` fo
 To connect to other servers, you might need to provision a different certificate.
 See :ref:`cert_dwload` for more information.
 
+Using MbedTLS and TF-M
+**********************
+
+This sample supports using MbedTLS and Trusted Firmware-M (TF-M).
+Instead of offloading the TLS sockets in to the modem, the MbedTLS library from Zephyr can be utilized.
+Using the Zephyr MbedTLS, the offloaded sockets utilizing the modem can still be used. The MbedTLS offers more
+configuration options than is possible when using the offloaded TLS handling.
+
+This sample uses the TF-M while using the MbedTLS, thus all the crypto operations are run on the secure side of the device.
```

Comment:
```suggestion
This sample uses TF-M and Mbed TLS.
Thus, all the crypto operations are run on the secure side of the device.
```

Reply:
In this section the TF-M is the prerequisite to run the crypto on the secure side, otherwise the crypto functionality of the MbedTLS would be used and ran on the unsecure side. Needs maybe a bit clarification here.

Reply:
Maybe this would be better
"While using TF-M and Mbed TLS using PSA crypto, all the crypto operations are run on the secure side on the device."

Reply:
Using x using y sounds a bit weird. How about:
When using TF-M and Mbed TLS with PSA crypto, ....?